dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```diff
@@ -11,12 +11,18 @@ jenkins:
       maxVirtualMachinesLimit: <%= @jcasc['cloud_agents']['azure-vm-agents']['maxInstances'] %>
       vmTemplates:
     <%- @jcasc['cloud_agents']['azure-vm-agents']['agent_definitions'].each do |agent| -%>
-      - launcher:
+      - agentWorkspace: "<%= @jcasc['agents_setup'][agent['os']]['agentDir'] %>"
+        <%- if agent['launcher'] && agent['launcher'] == "inbound" -%>
+        launcher: "inbound"
+        <%- else %>
+        launcher:
           ssh:
             preInstallSsh: false
-        agentWorkspace: "<%= @jcasc['agents_setup'][agent['os']]['agentDir'] %>"
+        launcher: "inbound"
```

Comment:
Typo?

Reply:
@dduportal to be removed? Seems odd to have this property defined twice.

Reply:
Good catch, thanks! https://github.com/jenkins-infra/jenkins-infra/pull/2927